src/components/Address.js

```diff
@@ -9,46 +9,50 @@ class Address extends React.Component {
     // flow:disable not supported yet
     static sampleProps = fixtures.ixa.location;
 
-    describe(property: string, description: string): React.Element {
-        var described = "";
+    describe(property: string, description: string): string {
```

Comment:
Add a docstring to this method? I thought it was related to `object.description` and got confused.